PiwikTracker/PiwikTracker.swift

```diff
@@ -38,10 +39,13 @@ final public class PiwikTracker: NSObject {
     ///   - siteId: The unique site id generated by the server when a new site was created.
     ///   - queue: The queue to use to store all analytics until it is dispatched to the server.
     ///   - dispatcher: The dispatcher to use to transmit all analytics to the server.
-    required public init(siteId: String, queue: Queue, dispatcher: Dispatcher) {
+    required public init(siteId: String, queue: Queue, dispatcher: Dispatcher, suiteName: String? = nil) {
```

Comment:
Hey @mattiaslevin, I just found another issue with this. If I would add this right away, we would add a suiteName to Trackers that currently don't have a suite name set. This would result in loosing their data.
So either we port the data over or we change the idea so the "old implementations" would keep their "no suite name" setting. I currently feel like migrating it over would be the best solution.